packages/pretty-format/src/plugins/lib/markup.js

```diff
@@ -9,7 +9,7 @@
 
 import type {Config, Printer, Refs} from 'types/PrettyFormat';
 
-import escapeHTML from './escape_html';
+import escapeHTML from './escapeHTML';
```

Comment:
Can you rename the rest of the files in pretty-format? There's quite a bunch of left

Reply:
Hey, I might have missed some, could you point out what files are left? I thought that the ones in `/src/` have multiple exports per file, hence I left files with snake_case

Reply:
We want to rename _every_ file that doesn't stick to the convention.
E.g. 
```
packages/pretty-format/src/__tests__/asymmetric_matcher.test.js
packages/pretty-format/src/plugins/asymmetric_matcher.js
```
to have names as variables:
```js
import AsymmetricMatcher from './plugins/asymmetric_matcher';
import ConvertAnsi from './plugins/convert_ansi';
import DOMCollection from './plugins/dom_collection';
import DOMElement from './plugins/dom_element';
import Immutable from './plugins/immutable';
import ReactElement from './plugins/react_element';
import ReactTestComponent from './plugins/react_test_component';
```

Reply:
But the `'./plugins/asymmetric_matcher'` has 3 exports, so according to those:

> All folders -> kabab-case
All files that export a single Class -> CapitalizeCase
All files that export a single function -> camelCase
All other files -> underscore_case

its falling into the `underscore_case`?

Reply:
This is a mistake in interface, they shouldn't expose `serialize` and `test` directly, as they do it in a default export – and we only use it that way. You can remove those exports. cc @pedrottimark

Reply:
@thymikee Great, I will redo the exports and fix those files, thanks 👍